cvat/apps/documentation/user_guide.md

```diff
@@ -254,102 +312,139 @@ Go to the first and latest frames.
 
 ![](static/documentation/images/image041.jpg)
 
-To adjust player speed go to settings (``Open Menu`` —> ``Settings``) and modify a value of ``Player Speed`` property.
+---
+Go to specified frame. Press ``~`` to highlight element.
 
-![](static/documentation/images/image042.jpg)
+![](static/documentation/images/image060.jpg)
 
-Go to specified frame.
+---
+__Open Menu__ button
 
-![](static/documentation/images/image060.jpg)
+It is the main menu for the annotation tool. It can be used to download, upload and remove annotations. As well it shows statistics about the current annotation task.
 
+![](static/documentation/images/image051.jpg)
 
-### Bottom side panel
+---
+__Filter__ input box
+
+__[All about Filters](./documentation/user_guide.html#filter)__
+
+![](static/documentation/images/image059.jpg)
+
+---
+__History / Undo-Redo panel__
+
+Use shortcuts for undo/redo actions ``Ctrl+Z`` __/__  ``Ctrl+Shift+Z``/``Ctrl+Y``
+
+![](static/documentation/images/image061.jpg)
 
-![](static/documentation/images/image043.jpg)
 
-### Side panel (list of objects)
+### Side panel
 
-In the side panel you can see the list of available objects on the current frame. An example how the list can look like below:
+#### Objects
+In the Side Panel you can see the list of available objects on the current frame. An example how the list can look like below:
 
 |Annotation mode|Interpolation mode|
 |--|--|
 |![](static/documentation/images/image044.jpg)|![](static/documentation/images/image045.jpg)|
 
----
+#### Labels
+You also can see all labels that used on this frame.
 
-A bounding box can be locked to prevent its modification or moving by an accident. Shortcut to lock an object: ``l``.
+![](static/documentation/images/image062.jpg)
 
-![](static/documentation/images/image046.jpg)
 
 ---
+__Objects' card__
+
 A bounding box can be removed. Shortcut: ``Delete``. A locked bounding box can be deleted using ``Shift+Delete`` shortcut.
 
 ![](static/documentation/images/image047.jpg)
 
+---
+A bounding box can be locked to prevent its modification or moving by an accident. Shortcut to lock an object: ``l``.
+
+![](static/documentation/images/image046.jpg)
+
 ---
 A bounding box can be **Occluded**. Shortcut: ``q``. Such bounding boxes have dashed boundaries.
 
 ![](static/documentation/images/image048.jpg)
 
 ![](static/documentation/images/image049.jpg)
 
+---
+You can copy and paste this object on this or other frame. ``Ctrl+C``/``Ctrl+V`` shortcuts works under mouse point.
+
+![](static/documentation/images/image052.jpg)
+
+---
+You can propagate this object on next X frames. ``Ctrl+B`` shortcut works under mouse point. ``F2`` for change on how many frames to propagate this object. 
+
+![](static/documentation/images/image053.jpg)
+
+---
+You can change this objects' annotation is displayed on this frame. It could be Hide, Shows Only Box, Shows Box and Title. ``H`` is for this object, ``T+H`` for all objects on this frame.
```

Comment:
This setting is independents by frame. Also: it could be Hide, Shows Only Shape, Shows Shape and Title